csrankings.csv

```diff
@@ -3592,6 +3593,7 @@ Edward Lank,University of Waterloo,https://cs.uwaterloo.ca/~lank,e7P8IwgAAAAJ
 Edward M. Reingold,Illinois Institute of Technology,https://science.iit.edu/people/faculty/edward-reingold,UlQeH7AAAAAJ
 Edward Omiecinski,Georgia Institute of Technology,http://www.cc.gatech.edu/~edwardo,NOSCHOLARPAGE
 Edward Sciore,Boston College,http://www.cs.bc.edu/~sciore,NOSCHOLARPAGE
+Edward Thomas Grefenstette,University College London,http://egrefen.com/,ezllEwMAAAAJ

```

Comment:
Please point to some source to back up the claim that he is not still at Deep Mind (which is what his home page states).

Reply:
@emeryberger https://twitter.com/egrefen/status/1077976894203662337

Reply:
@emeryberger [He joins FAIR](https://twitter.com/egrefen/status/1082264124543913984 ).

Reply:
@emeryberger fixed appropriately.